crates/ruff_python_ast/src/nodes.rs

```diff
@@ -3124,6 +3124,27 @@ impl Pattern {
             _ => false,
         }
     }
+
+    /// Checks if the [`Pattern`] is a [wildcard pattern].
+    ///
+    /// The following are wildcard patterns:
+    /// ```python
+    /// match subject:
+    ///     case _ as x: ...
+    ///     case _ | _: ...
+    ///     case _: ...
+    /// ```
+    ///
+    /// [wildcard pattern]: https://docs.python.org/3/reference/compound_stmts.html#wildcard-patterns
+    pub fn is_wildcard(&self) -> bool {
```

Comment:
Does this deserve tests of its own?

Reply:
Yeah, makes sense.

Reply:
Thanks, that did uncover a bug where the first case pattern in the above example wasn't being considered a wildcard.